library/testing/CMakeLists.txt

```diff
@@ -27,11 +23,21 @@ list(APPEND libf3dSDKTests_list
      TestPseudoUnitTest.cxx
 )
 
-# HDRI test needs https://gitlab.kitware.com/vtk/vtk/-/merge_requests/9767
-if(VTK_VERSION VERSION_GREATER_EQUAL 9.2.20221220)
+# Tests having UI widgets
+if(F3D_MODULE_UI)
```

Comment:
i guess its whatever test needing the filename, is that correct ?

Reply:
Indeed. And in the future there will be other widgets ofc.